ui/src/style/brandbar.less

```
@@ -41,13 +41,17 @@
   .brandbar__seperator {
     margin: -3px 10px;
 
-    border-left: 1px solid #D9DADB;
+    border-left: 1px solid #d9dadb;
   }
 
   .brandbar__s2-logo {
     margin-top: -4px;
     margin-right: 5px;
-    width: 28px;
+    height: 22px;
+
+    .yellow{fill:#F4D35E;}
+    .dark-blue{fill:#0F3875;}
+    .blue{fill:#1857B6;}
```

It's really up to you how clean you want this css, but I'd suggest a _bit_ cleaner than this. ;)
```suggestion
    .yellow { fill:#F4D35E; }
    .dark-blue { fill:#0F3875; }
    .blue { fill:#1857B6; }
```

I wonder if it's worth updating the class names in the SVG to be BEM style.